acai-ios/ViewControllers/LoginViewController.swift

```diff
@@ -79,58 +75,77 @@ class LoginViewController: UITableViewController {
         tableView.contentInset.bottom = FileConstants.inputViewBottomOffset
         tableView.register(InputTableViewCell.self, forCellReuseIdentifier: FileConstants.inputCellReuseIdentifier)
         tableView.register(RoundedButtonTableViewCell.self, forCellReuseIdentifier: FileConstants.roundedButtonCellReuseIdentifier)
+        tableView.register(TextTableViewCell.self, forCellReuseIdentifier: FileConstants.textCellReuseIdentifier)
     }
 
     private func formatNavigationBar() {
+        let navigationTitleLabel = UILabel()
+        navigationTitleLabel.text = "Account"
+        navigationTitleLabel.font = UIFont.avenirNextMedium.withSize(24)
+        navigationTitleLabel.sizeToFit()
+        let navigationBarTitleItem = UIBarButtonItem(customView: navigationTitleLabel)
+        let navigationBarBackButton = UIBarButtonItem(image: UIImage(named: "backArrow"), style: .done, target: self, action: #selector(backButtonPressed))
+        self.navigationItem.leftBarButtonItems = [
+            navigationBarBackButton,
+            navigationBarTitleItem
+        ]
         navigationController?.navigationBar.setBackgroundImage(nil, for: .default)
         navigationController?.navigationBar.setBackgroundImage(nil, for: .compact)
         navigationController?.navigationBar.shadowImage = UIImage()
         navigationController?.navigationBar.tintColor = .black
         navigationController?.navigationBar.barTintColor = .navigationWhite
+        navigationController?.navigationBar.prefersLargeTitles = false
         navigationController?.navigationBar.titleTextAttributes = [
             .font: UIFont.avenirNextMedium.withSize(24),
             .foregroundColor: UIColor.black
         ]
     }
 
+    @objc func backButtonPressed() {
+        navigationController?.popViewController(animated: true)
+    }
+
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         // +1 for cell with sign up and login buttons
-        return inputItems.count + 1
+        // +1 for label text
+        return inputItems.count + 2
     }
 
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        if indexPath.row == inputItems.count {
+        if indexPath.row == inputItems.count + 1 {
             let cell = tableView.dequeueReusableCell(withIdentifier: FileConstants.roundedButtonCellReuseIdentifier, for: indexPath) as! RoundedButtonTableViewCell
             cell.selectRoundedButtonDelegate = self
             return cell
+        } else if indexPath.row == 0 {
```

Comment:
nit: i think it makes it more readable if we had the case of `indexPath.row == 0 ` first